apps/condo/domains/news/schema/NewsItem.js

```diff
@@ -2,6 +2,9 @@
  * Generated by `createschema news.OrganizationNewsItem 'organization:Relationship:Organization:CASCADE; title:Text; body:Text; type:Select:common,emergency'`
  */
 
+const BadWordsNext = require('bad-words-next')
+const badWordsRu = require('bad-words-next/data/ru.json')
+const badWordsRuLat = require('bad-words-next/data/ru_lat.json')
```

Comment:
best to make it less dependable on number of languages

it would be great if you could only add specific .json in a specific folder and this filter will work

right now, you'd need to make some changes to the code

Reply:
These files are part of the package I used.
https://github.com/alexzel/bad-words-next/tree/main/data
This "filter" is only for 1st version of the feature. And will be improved in the future when specific requirements arise.

Reply:
Also, I tried to make fewer integrations within our code to easy remove this functionality